dandiapi/api/management/commands/extract_metadata.py

```diff
@@ -37,8 +41,12 @@ def get_asset_digest(asset: Asset) -> Digest:
 
 
 def extract_asset_metadata(asset: Asset, draft_version: Version):
+    # Test
+    s3_url = asset.s3_url.replace(
+        'http://localhost:9000/dandi-dandisets', 'https://dandiarchive.s3.amazonaws.com'
+    )
```

Comment:
Is this leftover from debugging?

Reply:
Whoops yes, thanks. That's the only good way I have to test the re-extraction.